ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -470,7 +471,20 @@ def gather_version_info_for_report(config):
 
         # add ceph version
         if not ocsci_config.ENV_DATA["mcg_only_deployment"]:
-            ceph_version = get_ceph_version()
+
```

Comment:
get_ceph_version uses tools pod which will be present in the provider cluster. The tools pods will be fetched from the provider cluster (when only one provider cluster is present in the contexts) and run command on the tools pod despite the current cluster context. If that is not working, it will be an automation issue that needs fix.

Reply:
This is the condition to use provider clusters's kubeconfig to get tools pod. 


https://github.com/red-hat-storage/ocs-ci/blob/fbf320b0382bb2ebc27acc3eade6ae699f6cd7ed/ocs_ci/ocs/resources/pod.py#L835-L846

Were these config parameters present in the test execution ?

Reply:
I think it is resolved now